Unresolved issues

jhgrng opened the pull request:
Hey! :wave: 

I resolved all of the found issues. One of them, ```Terasology```, was already added to the repo, but I *updated* its description(it was missing a dot at the end of the sentence), and it should get resolved due to [making a commit](https://github.com/jhgrng/games/commit/2d23310adaebc0b183186122cb1b1e5c48fd1762).

I was also thinking about *formatting* links' descriptions so that they all start with a capital letter, end with a dot, and so on. What do you think?

PS Trimming tab size from 8 spaces to 4 spaces was accidental, but I'm not really sure what is the convention behind tab size in Markdown - any suggestions? Since it's a ```markup language with plain text formatting syntax designed so that it can be converted to HTML and many other formats using a tool by the same name```(*via [Wikipedia](http://en.wikipedia.org/wiki/Markup_language)*), should it be 2 spaces? Thanks for helping!


Cheers! :smirk:

leereilly commented:
Hi :wave:

Thanks for all of these contributions! 

> I was also thinking about formatting links' descriptions so that they all start with a capital letter, end with a dot, and so on. What do you think?

I'm fine with either; as long as it's consistent :neckbeard: 

> PS Trimming tab size from 8 spaces to 4 spaces was accidental, but I'm not really sure what is the convention behind tab size in Markdown - any suggestions? Since it's a markup language with plain text formatting syntax designed so that it can be converted to HTML and many other formats using a tool by the same name(via Wikipedia), should it be 2 spaces? Thanks for helping!

No worries. I like it! 2 spaces would definitely be preferable per the [GFMD spec](https://help.github.com/articles/markdown-basics/#nested-lists). Do you want to change that? It'd help secure your 2nd place spot in the [contributions graph](https://github.com/leereilly/games/graphs/contributors) :trophy:

jhgrng commented:
@leereilly okie dokie, I [Changed tab size to 2 spaces](https://github.com/jhgrng/games/commit/a1fa59850991eb89720e81caab0cf0d94cf29fef)(thanks for the GFMD spec!!) as well as did a little [Descriptions cleanup](https://github.com/jhgrng/games/commit/cc0392de00b82d12da19aba2eb52846e9443e097) - it was mainly removing ```a```s and ```an```s from the beginning of the sentence and adding a dot at the end.


I also removed 2 duplicates(```A Dark Room``` and ```Spring``` RTS Game Engine), changed every ```Play now!``` to ```Play it now!```, as well as removed the phrase ```open source``` from every links' description - I just thought that since the first sentence of the project overview states that this is a ```list of open source games and game-related projects```, then stating that links' content is ```open source``` in links' description is unnecessary.

I now feel like I should've done the descriptions cleanup on a separate branch and create a separate pull request. Hope this isn't too much lines of code to review at once.

BTW how do you feel about about adding three back-ticks(```) to every technology related phrase, for example:
> * [Minecraft](https://github.com/fogleman/Minecraft) - Simple Minecraft-inspired demo written in [Python](https://www.python.org/) and [Pyglet](http://www.pyglet.org/).

would become:
> * [Minecraft](https://github.com/fogleman/Minecraft) - Simple Minecraft-inspired demo written in [```Python```](https://www.python.org/) and [```Pyglet```](http://www.pyglet.org/).

I was also thinking about adding links(if possible) to certain phrases in order for them to be more informative/helpful, like adding links to Wikipedia(for example: [*Minecraft*](http://en.wikipedia.org/wiki/Minecraft)) or link directly to technology's website(for example: [```Python```](https://www.python.org/)) - what do you think?


Thanks for the help! :ok_hand:
